crates/modelardb_server/src/storage/mod.rs

```diff
@@ -309,24 +309,23 @@ impl StorageEngine {
             .map_err(|error| error.to_string())
     }
 
-    /// Flush all of the data the [`StorageEngine`] is currently storing in memory to disk. If all
-    /// of the data is successfully flushed to disk, return [`Ok`], otherwise return [`String`].
+    /// Flush all the data the [`StorageEngine`] is currently storing in memory to disk. If all
+    /// the data is successfully flushed to disk, return [`Ok`], otherwise return [`String`].
     pub(super) async fn flush(&self) -> Result<(), String> {
         self.channels
             .multivariate_data_sender
             .send(Message::Flush)
             .map_err(|error| format!("Unable to flush data in storage engine due to: {}", error))?;
 
-        // Wait until all of the data in the storage engine have been flushed.
+        // Wait until all the data in the storage engine have been flushed.
```

Comment:
I think we used "data" in singular form earlier (also in l 313 above), so have -> has for consistency

Reply:
Fixed.